src/components/ec-datepicker/ec-datepicker.vue

```diff
@@ -127,6 +134,7 @@ export default {
     if (this.flatpickrInstance.calendarContainer) {
       this.flatpickrInstance.calendarContainer.dataset.test = 'ec-datepicker__calendar';
       this.flatpickrInstance.calendarContainer.dataset.relDataTest = `${this.$attrs['data-test']} ec-datepicker`.trim();
+      this.flatpickrInstance.calendarContainer.classList.add(`flatpickr-calendar--${this.level}`);
```

Comment:
Depends, I added as a default value 'level-1' if we keep it, I do not think is worth testing because will always be there, if we agree that should not have an actual default value, then it can be empty, then we can test it that might exist/not-exist